llvmlite/tests/test_ir.py

```diff
@@ -111,6 +111,15 @@ def test_define(self):
 
 class TestIR(TestBase):
 
+    def test_globals_access(self):
+        mod = self.module()
+        ir.Function(mod, ir.FunctionType(ir.VoidType(), []), 'foo')
+        ir.Function(mod, ir.FunctionType(ir.VoidType(), []), 'bar')
+        ir.GlobalVariable(mod, ir.DoubleType(), 'globdouble')
+        self.assertIsInstance(mod.get_global('foo'), ir.Function)
```

Comment:
Can't you just test that the object created above is returned? Or am I missing something?